packages/node-cli/src/core/CommandActionExecutor.js

```diff
@@ -68,11 +67,12 @@ module.exports = class CommandActionExecutor {
 				commandUserExtension: commandUserExtension,
 				projectConfiguration: projectConfiguration,
 			});
-
 			if (actionResult.isSuccess() && commandUserExtension.onCompleted) {
 				commandUserExtension.onCompleted(actionResult);
-			}
-			else if (!actionResult.isSuccess() && commandUserExtension.onError) {
+				if (context.runInInteractiveMode) {
+					this._showNonInteractiveCommand(commandName, commandMetadata, actionResult);
+				}
```

Comment:
If we deploy a project that fails because of the server-side validation we won't show the non-interactive equivalence . We should ask Carlos about this case and other possible similar cases.